todo-app/src/app.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { TodoModule } from './todo/module';
+
+@Module({
+  imports: [
+    TodoModule,
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: 'localhost',
+      port: 5432,
+      username: 'postgres',
+      password: 'postgres',
+      database: 'todo-app',
+      autoLoadEntities: true,
+      synchronize: true,
```

Comment:
- do you know what this line does? if not, read up on it further and let's discuss then